devsetup/Makefile

```diff
@@ -48,6 +48,7 @@ EDPM_NETWORKER_SUFFIX ?= 0
 EDPM_COMPUTE_ADDITIONAL_NETWORKS ?= '[]'
 EDPM_TOTAL_NODES ?= 1
 EDPM_COMPUTE_CEPH_ENABLED ?= true
+EDPM_COMPUTE_CEPH_NOVA ?= true
```

Comment:
@SeanMooney It seems we are still having the same chicken end egg problem as we cannot have the 'true' path enabled here, because ithatwould create a circular dependency on the adoption pr.

Reply:
nevermind, I got how to make it passing for now
https://review.rdoproject.org/r/c/rdo-jobs/+/52931